internal/elasticsearch/api/search.go

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2024 Meroxa, Inc. and Miquido
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+// SearchRequest is the request for calling ElasticSearch api.
+type SearchRequest struct {
+	Index       string `json:"index"`
+	Size        *int   `json:"size:"`
+	SearchAfter int64  `json:"searchAfter"`
+	SortBy      string `json:"sortBy"`
+	Order       string `json:"order"`
+}
```

Comment:
Can we use _seq_no instead? Are there any performance implications?

Reply:
Okay, let's fix it using _seq_no. This way config also becomes clean and simpler.